src/js/components/Diagram/stories/Progressing.js

```diff
@@ -51,7 +51,11 @@ export const Progressing = () => {
 
   if (topRow.length >= 3) {
     connections.push(
-      connection('3', '5', { anchor: 'horizontal', color: 'brand' }),
+      connection('3', '5', {
```

Comment:
Let's add a better description to the title of the story, explaining to the user about the animation that is being used. one sentence should be sufficient.